auto_gromos/ff_writers.py

```diff
@@ -0,0 +1,81 @@
+import networkx as nx
+
+def write_blocks(blocks, outname):
+    """
+    Write `blocks` in a file `outname`
+    using vermouth format.
+
+    Parameters:
+    -----------
+    blocks: list[:class:`vermouth.molecule.blocks`]
+    outname: str
+    """
+    with open(outname, "w") as _file:
+        for block in blocks:
+           # write header
+            _file.write("[ moleculetype ]\n")
+            name = nx.get_node_attributes(block, "resname")[0]
+            _file.write(name + " 3\n")
+            _file.write("[ atoms ]\n")
+
+            # write atom section
+            for node in block.nodes:
+                atom = block.nodes[node]
+                _format_atoms = " ".join(['{}' for _ in atom.keys()]) + "\n"
+                _line_values = [node] + [atom[key] for key in ["atype", "resid",
+                                                               "resname", "atomname",
+                                                               "charge_group", "charge", "mass"]]
```

Comment:
```suggestion
                attributes = ["atype", "resid", resname", "atomname", charge_group", "charge", "mass"]
                _format_atoms = " ".join(['{}']*(len(attributes) + 1)) + "\n"
                _line_values = [node] + [atom[key] for key in attributes]
```
With some effort you could also make it a named format string, and do `**nodes` instead of `*_line_values`. Not sure what that gains you though